lib/pronto/config_file.rb

```
@@ -35,8 +36,12 @@ class ConfigFile
       'format' => DEFAULT_MESSAGE_FORMAT
     }.freeze
 
-    def initialize(path = '.pronto.yml')
-      @path = path
+    def initialize
+      @path = if File.exist?(ENV['PRONTO_CONFIG'].to_s)
```

I think we should raise an error and not silently use the default:

```ruby
def initialize
  @path = ENV.fetch('PRONTO_CONFIG', DEFAULT_FILE_PATH) 
  raise "configuration file `#{@path}` missing" unless File.exists?(@path)
end
```